examples/fundamentals.test.js

```diff
@@ -4355,6 +4355,34 @@ describe('Fundamentals', function() {
       // acquit:ignore:end
     });
   });
+    it('NaNs', function() {
+      isNaN(NaN); //true
+      isNaN(Number.NaN); //true
+      Number.isNaN(NaN); //true
+      let text = 'hello world!';
+      if (isNaN(text)) {
```

Comment:
Instead of using `if` statements with `console.log()`, I'd just do something like `isNaN(text); // true`